unittests/CMakeLists.txt

```diff
@@ -27,3 +27,6 @@ add_test( NAME test_tracing_node COMMAND python ${CMAKE_CURRENT_LIST_DIR}/test_t
 add_test( NAME test_reward_synapse COMMAND python ${CMAKE_CURRENT_LIST_DIR}/test_reward_synapse.py )
 add_test( NAME test_reward_synapse_io COMMAND python ${CMAKE_CURRENT_LIST_DIR}/test_reward_synapse_io.py )
 add_test( NAME test_reward_in_proxy COMMAND python ${CMAKE_CURRENT_LIST_DIR}/test_reward_in_proxy/test.py )
+
+# Integration Tests
+add_test( NAME test_pattern_matching COMMAND python ${CMAKE_CURRENT_LIST_DIR}/test_pattern_matching.py )
```

Comment:
Plus, we do not really test whether pattern matching works (just if the script runs). Running a full learning experiment as a test would be kind of nice because it would *really* show that SPORE works, but the overhead is not practical I guess.